cmd/admin-user-add.go

```diff
@@ -133,16 +136,18 @@ func (u userMessage) JSON() string {
 }
 
 // fetchUserKeys - returns the access and secret key
```

Comment:
Should we rename the function then and change comment or should we put handling of additional arguments for mapped user/group into separate function?

Reply:
Renamed